sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -64,9 +65,37 @@
   String PROP_OWNER_TYPE = "ownerType";
 
   /**
-   * The list of reserved namespace properties.
+   * The list of namespace ownership properties, which can be used in `ALTER` syntax:
+   *
+   * {{
+   *   ALTER (DATABASE|SCHEMA|NAMESPACE) SET OWNER ...
+   * }}
+   */
+  List<String> OWNERSHIPS = Arrays.asList(PROP_OWNER_NAME, PROP_OWNER_TYPE);
+
+  /**
+   * The list of immutable namespace properties, which can not be removed or changed directly by
+   * the syntax:
+   * {{
+   *   ALTER (DATABASE|SCHEMA|NAMESPACE) SET DBPROPERTIES(...)
+   * }}
+   *
+   * They need specific syntax to modify
+   */
+  List<String> REVERSED_PROPERTIES =
+    Arrays.asList(PROP_COMMENT, PROP_LOCATION, PROP_OWNER_NAME, PROP_OWNER_TYPE);
+
+  /**
+   * Specify the default owner name for `CREATE` namespace.
+   *
+   */
+  default String defaultOwner() { return Utils.getCurrentUserName(); }
```

Comment:
Can we do it later? I don't think there is a consensus yet. For now let's let spark decide the default owner, as how we do it for tables in hive metastore.

Reply:
OK